specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+        "result": {
+          "$ref": "#/definitions/RunNotebookSnapshotResult"
+        }
+      }
+    },
+    "RunNotebookSparkSessionOptions": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "The session kind"
+        },
+        "proxyUser": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "User to impersonate when starting the session"
+        },
+        "name": {
+          "x-nullable": true,
+          "type": "string",
+          "description": "The name of this session"
+        },
+        "jars": {
+          "uniqueItems": false,
+          "x-nullable": true,
+          "type": "array",
+          "description": "jars to be used in this session",
+          "items": {
+            "type": "string"
+          }
+        },
+        "pyFiles": {
+          "x-ms-client-name": "pythonFiles",
+          "uniqueItems": false,
+          "x-nullable": true,
+          "description": "Python files to be used in this session",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "files": {
+          "uniqueItems": false,
+          "type": "array",
+          "x-nullable": true,
+          "description": "files to be used in this session",
+          "items": {
+            "type": "string"
+          }
+        },
+        "archives": {
+          "uniqueItems": false,
+          "x-nullable": true,
+          "description": "Archives to be used in this session",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "queue": {
+          "type": "object",
+          "x-nullable": true,
+          "description": "The name of the YARN queue to which submitted"
+        },
+        "conf": {
+          "x-ms-client-name": "configuration",
+          "type": "object",
+          "description": "Spark configuration properties",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "driverMemory": {
+          "type": "string",
+          "description": "Amount of memory to use for the driver process"
+        },
+        "driverCores": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of cores to use for the driver process"
+        },
+        "executorMemory": {
+          "type": "string",
+          "description": "Amount of memory to use per executor process"
+        },
+        "executorCores": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Number of cores to use for each executor"
+        },
+        "numExecutors": {
+          "x-ms-client-name": "executorCount",
```

Comment:
Here too -- please consider making this change in the API.

Reply:
We cannot change the property name because it is open source LIVY standard. https://livy.incubator.apache.org/docs/latest/rest-api.html